x-pack/plugin/analytics/src/main/java/org/elasticsearch/xpack/analytics/AnalyticsPlugin.java

```diff
@@ -123,7 +124,9 @@ public List<AggregationSpec> getAggregations() {
                 RateAggregationBuilder.NAME,
                 RateAggregationBuilder::new,
                 usage.track(AnalyticsStatsAction.Item.RATE, RateAggregationBuilder.PARSER)
-            ).addResultReader(InternalRate::new).setAggregatorRegistrar(RateAggregationBuilder::registerAggregators),
+            ).addResultReader(InternalRate::new)
```

Comment:
Should this for now only be registered when time series mode is enabled? (by adding a `IndexSettings.isTimeSeriesModeEnabled()` if statement)